src/app/i18n/zh-CN.ts

```diff
@@ -4,278 +4,283 @@
-        <location filename="../../ui/src/abstractlogview.cpp" line="1837"/>
+        <location filename="../../ui/src/abstractlogview.cpp" line="1868"/>
         <source>Send to scratchpad</source>
-        <translation type="unfinished"></translation>
+        <translation>追加到暂存器</translation>
     </message>
     <message>
-        <location filename="../../ui/src/abstractlogview.cpp" line="1841"/>
+        <location filename="../../ui/src/abstractlogview.cpp" line="1872"/>
         <source>Replace scratchpad</source>
-        <translation type="unfinished"></translation>
+        <translation>替换暂存器</translation>
     </message>
 </context>
 <context>
     <name>CrawlerWidget</name>
     <message>
-        <location filename="../../ui/src/crawlerwidget.cpp" line="414"/>
+        <location filename="../../ui/src/crawlerwidget.cpp" line="419"/>
         <source>klogg</source>
-        <translation type="unfinished"></translation>
+        <translation>klogg</translation>
     </message>
     <message>
-        <location filename="../../ui/src/crawlerwidget.cpp" line="415"/>
+        <location filename="../../ui/src/crawlerwidget.cpp" line="420"/>
         <source>Search history:</source>
-        <translation type="unfinished"></translation>
+        <translation>搜索历史：</translation>
     </message>
     <message>
-        <location filename="../../ui/src/crawlerwidget.cpp" line="471"/>
+        <location filename="../../ui/src/crawlerwidget.cpp" line="477"/>
         <source>Search in progress (%1 %)...</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished">正在搜索（%1 %）...</translation>
     </message>
     <message>
-        <location filename="../../ui/src/crawlerwidget.cpp" line="472"/>
+        <location filename="../../ui/src/crawlerwidget.cpp" line="478"/>
         <source> %1 matches found so far.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished"> 已找到 %1 的匹配.</translation>
     </message>
     <message>
-        <location filename="../../ui/src/crawlerwidget.cpp" line="474"/>
+        <location filename="../../ui/src/crawlerwidget.cpp" line="480"/>
         <source> %1 match found so far.</source>
-        <translation type="unfinished"></translation>
+        <translation type="unfinished"> 已找到 %1 的匹配.</translation>
     </message>
     <message>
-        <location filename="../../ui/src/crawlerwidget.cpp" line="944"/>
+        <location filename="../../ui/src/crawlerwidget.cpp" line="952"/>
         <source>Marks and matches</source>
-        <translation type="unfinished"></translation>
+        <translation>标记和匹配</translation>
     </message>
     <message>
-        <location filename="../../ui/src/crawlerwidget.cpp" line="949"/>
+        <location filename="../../ui/src/crawlerwidget.cpp" line="957"/>
         <source>Marks</source>
-        <translation type="unfinished"></translation>
+        <translation>仅标记</translation>
```

Comment:
```suggestion
        <translation>标记</translation>
```

Reply:
The options in the context include "marks and match, marks, match",  so I think it's clearer to translate it as "marks only" here